courses/admin.py

```diff
@@ -85,6 +89,10 @@ class CourseRunEnrollmentAdmin(admin.ModelAdmin):
     """Admin for CourseRunEnrollment"""
 
     model = CourseRunEnrollment
+    search_fields = ["user", "company", "CourseRun"]
```

Comment:
This is going to fail for the following reasons:
- `user`, `company`, `CourseRun` are all foreign key fields, so the admin will fail as soon as you do a text search on the changelist view.
- `CourseRun` should be changed to `run` as that is the correct attribute name.
- `user` should be changed to either `user__email` or `user__name` whichever is required. Same goes for `company => company__name` and `run => run__title`